src/main/java/ch/uzh/ifi/hase/soprafs22/game/Game.java

```diff
@@ -79,7 +82,20 @@ public GameMap getGameMap() {
     }
 
     public int nextTurn() {
-        return ++turnNumber;
+        // TODO test for real once turns can finish through the web.
+        ++turnNumber;
+        for (IPlayer player : playerMap.values()) {
+            String token = player.getToken();
+            if (isPlayersTurn(token))
+                socketMessage.convertAndSend("/topic/game/token" + token, "yes_turn");
```

Comment:
I would send back the id, not the token. The other users shouldn't have access to the token of other users.

Reply:
maybe we can send the id of the player who has to play instead of conditionally send "yes"/"no". Then, the client will update in the localStorage a variable which indicates the turn.

Reply:
Sorry Alessio XD I didn't see your comment. Yes, I agree with you :)

Reply:
Sounds good, done.